etl/steps/data/grapher/climate/2024-11-19/total_precipitation_annual.meta.yml

```diff
@@ -0,0 +1,44 @@
+definitions:
```

Comment:
I think it's more maintainable if you keep two tables in garden and call the yearly data in grapher. You will be able to use the metadata for both tables (description from producer, processing, etc)